Tabloid/Models/ViewModels/PostTagViewModel.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc.Rendering;
+using Microsoft.Extensions.Hosting;
+using System.Collections.Generic;
+using System.Linq;
+using Tabloid.Models;
+
+namespace Tabloid.Models.ViewModels
+{
+    public class PostTagViewModel
```

Comment:
What is this for? It doesn't look like it's referenced by any of the other code.